src/views/partials/results/template.njk

```diff
@@ -116,6 +117,9 @@
       </div>
     {% endfor %}
   </div>
+
+{{ govukPagination(paginationItems) }}
```

Comment:
I believe we need to add 20 padding at the top of the pagination block to create some space between the last document and the pagination. Can you please check the screen in the PR description and the Figma?